task_1.py

```diff
@@ -0,0 +1,6 @@
+num=15
+num=num + 5
+num=num - 3
+num=num * 4
+num=num / 2
+print('num',int(num))
```

Comment:
Do not forget about spaces, we use them to make code more clear and readable
```suggestion
num = 15
num = num + 5
num = num - 3
num = num * 4
num = num / 2
print('num', int(num))
```